react/scss/composers-app.scss

```diff
@@ -0,0 +1,31 @@
+@import '~bootstrap/dist/css/bootstrap.css';
+
+.pagination__header, .pagination__search, .pagination__suggestions, .pagination__content, .pagination__footer {
+  margin: 20px 0 20px 0;
+}
+
+.pagination-footer__separator {
```

Comment:
pagination-footer__separator is a subcomponent __separator that belongs to pagination-footer. pagination-footer is the main component.

pagination__footer__separator would mean that there is a component __footer that has entity __separator IN pagination component. (but it is actually a __separator in pagination-footer component). If we did pagination__footer__separator, then the pagination-footer would need to assume that it belongs to pagination component and I don't like assumptions. If they are in different files, I separate them as own entities. pagination__footer__separator couples pagination on footer and separator. pagination-footer__separator couples separator on pagination-footer.